### PR TITLE
Add independent Dockerfile for GHCR

### PR DESCRIPTION
**Pull request by dottorblaster**
This PR:

- Adds an independent dockerfile for the building and pushing to GHCR
- Removes registry.suse.com prefix in OBS-synced Dockerfile

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+LABEL org.opencontainers.image.source="https://github.com/trento-project/checks"
+
+# If set to C, LC_ALL takes precedence
+ENV LC_ALL=C.UTF-8
+
+# tar is required by kubectl cp
+RUN zypper --non-interactive in -y tar && \
+    zypper --non-interactive clean
+
```
Comment (arbulu89) [inline Dockerfile:11]: nitpick: (odc). I would remove the double new lines that are across the file. They make it the file look "not cared"

**Review by arbulu89 — APPROVED** [review]
:+1:

File: Dockerfile (hunk 1 above)
Comment (dottorblaster) [reply Dockerfile:11]: You are actually right

Comment (arbulu89) [reply Dockerfile:11]: Thank you! It looks more professional (to me at least XD)

**Pull request merged**